lecli/api_key/commands.py

```diff
@@ -0,0 +1,67 @@
+import json
+
+import click
+import sys
+
+from lecli.api_key import api
+
+
+@click.command()
+@click.argument('api_key', type=click.STRING)
+def get_api_key(api_key):
+    """
+    Get a specific api key
+    """
+    api.get(api_key)
+
+
+@click.command()
+def get_api_keys():
+    """
+    Get all api keys
+    """
+    api.list()
+
+
+@click.command()
+@click.argument('filename', type=click.Path(exists=True, dir_okay=False))
+def create_api_key(filename):
+    """
+    Create an api key with the provided file.
+    """
+    if filename is not None:
+        with open(filename) as json_data:
+            try:
+                params = json.load(json_data)
+                print params
+                api.create(params)
+            except ValueError as error:
```

Comment:
This should only be wrapped around the one line, `params = json.load(json_data)` right?
I don't believe api.create will throw that exception, or will it?